glean/rts/factset.cpp

```diff
@@ -260,7 +259,7 @@ Id FactSet::define(Pid type, Fact::Clause clause, Id) {
 
 thrift::Batch FactSet::serialize() const {
   binary::Output output;
-  for (auto& fact : *this) {
+  for (auto fact : *this) {
```

Comment:
Would this create another copy, wouldn't it?

Reply:
This is a `Fact::Ref` now (rather than a `Fact::unique_ptr`), those are supposed to be used by-copy.